integration/testcmd/gotest/uinit/gotest.go

```diff
@@ -37,7 +40,37 @@ func walkTests(testRoot string, fn func(string, string)) error {
 	})
 }
 
+// AppendFile takes two filepaths and concatenates the files at those.
+func AppendFile(srcFile, targetFile string) error {
```

Comment:
This should be in a uroot package at some point, it's not the only place it will be needed.